src/index.js

```diff
@@ -2156,6 +2223,20 @@ class Compute extends common.Service {
   image(name) {
     return new Image(this, name);
   }
+  /**
+   * Get a reference to a Google Compute Engine instance template.
+   *
+   * @see [Instance Templates Overview]{@link https://cloud.google.com/compute/docs/instanceTemplates}
+   *
+   * @param {string} name - Name of the instance template.
+   * @returns {Image}
+   *
+   * @example
+   * var instanceTemplate = gce.instanceTemplate('instance-template-name');
+   */
+  instanceTemplate(name) {
+    return new InstanceTemplate(this, name);
```

Comment:
Could you add a `throw(new Error('An instance template name is required.'))` here if `name` isn't provided?

Reply:
Fixed